curriculum/challenges/english/14-responsive-web-design-22/learn-css-colors-by-building-a-set-of-colored-markers/619b7c3c83de403126b69c1e.md

```diff
@@ -15,7 +15,9 @@ You may already be familiar with decimal, or base 10 values, which go from 0 - 9
 
 With hex colors, `00` is 0% of that color, and `FF` is 100%. So `#00FF00` translates to 0% red, 100% green, and 0% blue, and is the same as `rgb(0, 255, 0)`.
 
-Lower the intensity of green by setting green value of the hex color to `7F`.
+Remember the case of the `A-F` that hex colors take doesn't matter. But let's stick to using capital letters `A-F`. Don't mix up the cases in your future CSS projects.
```

Comment:
Same comment as before – we shouldn't need to explain this. We should just ensure that the projects' tests will pass with either lowercase or capital letters.

Reply:
I've removed that whole line now